vllm/transformers_utils/tokenizer_group/tokenizer_group.py

```diff
@@ -57,7 +59,11 @@ def encode(self,
                request_id: Optional[str] = None,
                lora_request: Optional[LoRARequest] = None) -> List[int]:
```

Comment:
To be consistent with HF, I think `add_special_tokens` should be an argument to `encode` and `encoder_async` instead of the initializer.